templates/types/llamaindexserver/fastapi/main.py

```diff
@@ -14,33 +12,17 @@
 
 
 def create_app():
-    env = os.environ.get("APP_ENV")
-
     app = LlamaIndexServer(
         workflow_factory=create_workflow,  # A factory function that creates a new workflow for each request
         ui_config=UIConfig(
             component_dir=COMPONENT_DIR,
             app_title="Chat App",
         ),
-        env=env,
         logger=logger,
     )
-    # You can also add custom routes to the app
```

Comment:
how about keep this so user know they can add custom routes?
but clarify that its FastAPI:
```You can also add custom FastAPI routes to app````